docs/google/best_practices.md

```diff
@@ -129,9 +129,19 @@ One word of caution is there is a limit to the total number of repetitions.  Tak
 that your parameter sweeps, especially products of sweeps, do not become so excessively large
 that they overcome this limit.
 
+## Use batches if sweeps are not possible
+
+The engine has a method called `run_batch()` that can be used to send multiple
+circuits in a single request.  This can be used to increase the efficiency
+of your program so that more repetitions are completed per second.
+
+In order to efficiently batch circuits, the circuits in these batches must
```

Comment:
```suggestion
The circuits that are meant to be executed in batches must
```
I.e. there is no such thing as "efficiently batching" and "less efficiently batching" - it seems that this is more binary - you must measure the same qubits otherwise there is no batching.